### DISCUSSION TITLE
How to send 'Reason' field when using CloudFormation Custom resource

### DISCUSSION THREAD

**Discussion by rahuldeverani** [aws-powertools/powertools-lambda-java#1742] [Q&A]
Hi team , I am following this doc: https://docs.powertools.aws.dev/lambda/java/utilities/custom_resources/#customising-a-response , to customise custom resource response.

I want my CloudFormation stack to show why the custom resource failed. For that we can send 'Reason' in the CloudFormation response :  https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/crpg-ref-responses.html

How to send this 'Reason' using lambda powertool custom resource implementation ?

**Comment by phipag** [comment]
Hi @rahuldeverani, I moved your question to issue https://github.com/aws-powertools/powertools-lambda-java/issues/1801. Please see my response there.
